packages/connect-form/src/components/connect-form.tsx

```diff
@@ -48,10 +48,18 @@ const descriptionStyles = css({
   marginTop: spacing[2],
 });
 
+const scrollbarWidth = spacing[2];
 const formContentContainerStyles = css({
   padding: spacing[4],
-  overflow: 'scroll',
+  paddingRight: spacing[4] - scrollbarWidth,
+  overflowY: 'scroll',
   position: 'relative',
+  scrollPadding: 0,
```

Comment:
This is not scrollbar padding, it's a padding that browser considers a viewbox for the scollable content when scroll snap is used (it also almost always 0 by default anyway), we can probably remove that, unless I'm missing something
```suggestion
```

Reply:
this is leftover from some brute-force attempts that ended in a git stash by mistake :)